.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+script: 
+- ./gradlew build
+sudo: required
+jdk:
+  - oraclejdk8
```

Comment:
Added these suggested configs. Seems like the second one worked, even with sudo: required
```
Installing APT Packages (BETA)
$ export DEBIAN_FRONTEND=noninteractive
$ sudo -E apt-get -yq update &>> ~/apt-get-update.log

$ sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install oracle-java8-installer
```